core/src/main/java/com/github/szsalyi/flashcardapp/entity/Language.java

```diff
@@ -0,0 +1,37 @@
+package com.github.szsalyi.flashcardapp.entity;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+/**
+ * Created by ssalyi on 8/14/2017.
+ */
+public class Language implements Serializable {
```

Comment:
Missing `@Entity`.
Rename the class to something like `LanguageEntity` to distinguish it from domain objects of upper layers.
Use the `@Table` annotation to specify the name of the database table for this entity (like 'languages').

Use Lombok to generate boilerplate code.